solr/solrj/src/test/org/apache/solr/client/solrj/request/TestCoreAdmin.java

```diff
@@ -80,7 +80,7 @@ public void testCustomUlogDir() throws Exception {
       cores.getAllowPaths().add(dataDir.toPath()); // Allow the test dir
       cores.getAllowPaths().add(newCoreInstanceDir.toPath()); // Allow the test dir
 
-      File instanceDir = new File(cores.getSolrHome());
+      File instanceDir = cores.getSolrHome().toFile();
```

Comment:
Could we stay in path land if we change FileUtils?

Reply:
Yes, it's possible to replace FileUtils here with `org.apache.commons.io.file.PathUtils`. For now I changed just the callsites for CoreContainer#getSolrHome, to simplify the review.